search/docker-compose-dev.yml

```diff
@@ -0,0 +1,121 @@
+version: "3"
+services:
+  # nginx reverse proxy
+  reverse-proxy:
+    image: reverse-proxy
+    container_name: reverse_proxy
+    depends_on:
+        - controller
+        - reactivesearch
+        - kibana
+    build:
+        context: ./nginx/nginx
+    ports:
+      - "80:80"
+    networks:
+      - elastic
+
+  # django controller
+  controller:
```

Comment:
I find the name "controller" a bit surprising here. Could we come up with a better one? backend? django? api?

Reply:
ok, I'll put it as backend